File: Domain/EmployeeTraining.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Domain
{
    public class EmployeeTraining
    {
        [Key]
        public string EmployeeNumber { get; set; }

        public Boolean HSTCbxSSLeader { get; set; }
        public Boolean HSTCbxRFSafety { get; set; }
        public Boolean HSTCbxOsha10Safety { get; set; }
        public Boolean HSTCbxOsha24HazWoper { get; set; }
        public Boolean HSTCbxOsha40HazWoper { get; set; }

        public Boolean HSTCbxOsha8HazWoper { get; set; }
        public Boolean HSTCbxFProtection { get; set; }
        public Boolean HSTCbxCSpace { get; set; }
        public Boolean HSTCbxASLift { get; set; }
        public Boolean HSTCbxAHSCTraining { get; set; }

        public Boolean HSTCbxCprFirstAid { get; set; }
        public Boolean HSTCbxHProtection { get; set; }
        public Boolean HSTCbxESurveyors { get; set; }
        public Boolean HSTCbxRSManagers { get; set; }
        public Boolean HSTCbxOther { get; set; }

        [StringLength(40)]
        public string HSTTxtOther { get; set; }

        public Boolean BSTTCbxPInitiation { get; set; }
        public Boolean BSTTCbxAPayable { get; set; }
        public Boolean BSTTCbxARCllections { get; set; }
        public Boolean BSTTCbxBilling { get; set; }
        public Boolean BSTTCbxIReports { get; set; }
        public Boolean BSTTCbxPortals { get; set; }

        public Boolean TDCbxCStandards { get; set; }
        public Boolean TDCbxSWorkflow { get; set; }
        public Boolean TDCbxSurfaces { get; set; }
        public Boolean TDCbxAlignments { get; set; }
        public Boolean TDCbxProfiles { get; set; }
        public Boolean TDCbxPNetworks { get; set; }
        public Boolean TDCbxCSections { get; set; }
        public Boolean TDCbxParcels { get; set; }

    }
}

File: client-app/src/app/models/employeeTraining.ts
export interface IEmployeeTraining {

    employeeNumber: string;

    hstCbxSSLeader: boolean;
    hstCbxRFSafety: boolean;
    hstCbxOsha10Safety: boolean;
    hstCbxOsha24HazWoper: boolean;
    hstCbxOsha40HazWoper: boolean;

    hstCbxOsha8HazWoper: boolean;
    hstCbxFProtection: boolean;
    hstCbxCSpace: boolean;
    hstCbxASLift: boolean;
    hstCbxAHSCTraining: boolean;

    hstCbxCprFirstAid: boolean;
    hstCbxHProtection: boolean;
    hstCbxESurveyors: boolean;
    hstCbxRSManagers: boolean;
    hstCbxOther: boolean;

    hstTxtOther: string;

    bsttCbxPInitiation: boolean;
    bsttCbxAPayable: boolean;
    bsttCbxARCllections: boolean;
    bsttCbxBilling: boolean;
    bsttCbxIReports: boolean;
    bsttCbxPortals: boolean;

    tdCbxCStandards: boolean;
    tdCbxSWorkflow: boolean;
    tdCbxSurfaces: boolean;
    tdCbxAlignments: boolean;
    tdCbxProfiles: boolean;
    tdCbxPNetworks: boolean;
    tdCbxCSections: boolean;
    tdCbxParcels: boolean;
}

export class EmployeeTrainingFormValues implements IEmployeeTraining {

    employeeNumber: string = '';

    // Health & Safety Training Checkbox Group
    hstCbxSSLeader: boolean = false; 
    hstCbxRFSafety: boolean = false; 
    hstCbxOsha10Safety: boolean = false; 
    hstCbxOsha24HazWoper: boolean = false; 
    hstCbxOsha40HazWoper: boolean = false;
    hstCbxOsha8HazWoper: boolean = false; 
    hstCbxFProtection: boolean = false; 
    hstCbxCSpace: boolean = false; 
    hstCbxASLift: boolean = false; 
    hstCbxAHSCTraining: boolean = false;
    hstCbxCprFirstAid: boolean = false; 
    hstCbxHProtection: boolean = false; 
    hstCbxESurveyors: boolean = false; 
    hstCbxRSManagers: boolean = false; 
    hstCbxOther: boolean = false; 
    hstTxtOther: string = '';

    // BST Training Checkbox Group
    bsttCbxPInitiation: boolean = false; 
    bsttCbxAPayable: boolean = false; 
    bsttCbxARCllections: boolean = false;
    bsttCbxBilling: boolean = false; 
    bsttCbxIReports: boolean = false; 
    bsttCbxPortals: boolean = false;

    // Technical & Design Checkbox Group
    tdCbxCStandards: boolean = false; 
    tdCbxSWorkflow: boolean = false; 
    tdCbxSurfaces: boolean = false; 
    tdCbxAlignments: boolean = false;
    tdCbxProfiles: boolean = false; 
    tdCbxPNetworks: boolean = false; 
    tdCbxCSections: boolean = false; 
    tdCbxParcels: boolean = false;
}
